### DISCUSSION TITLE
Possibility of adding more weightage for some atoms in the loss function

### DISCUSSION THREAD

**Discussion by kousi12** [ACEsuit/mace#1100] [Q&A]
Hello,

Is it possible to add more weightage for some atoms in the loss function during training from scratch and also finetuning in MACE?

**Comment by ilyes319** [comment]
That is not currently supported unfortunately, but it is a feature we want to add. We do have weights per structure/per type of configurations.
